README.md

```diff
@@ -557,12 +591,262 @@ end
 
 ---------------------------------------------
 
-![Interactive Tile 2 Output](https://github.com/alinchis/Escher.jl/blob/master/assets/img/interactive_tile_2.png "Interactive Tile 2")
+![Interactive Tile 2 Output](./assets/img/interactive_ui.png "Interactive Tile 2")
 
 ---------------------------------------------
 
 ##### 4.2.5.3. Interactive Tiles
 
 Escher provides ready-to-use `Tiles` like buttons, input, codemirror etc., in the [Widgets API](http://escher-jl.org/widgets-api.html). Keep in mind that, regular and interactive, `Tiles` can be augmented with clickable, selectable, keypress etc. behavior via the [Behavior API](http://escher-jl.org/behavior-api.html).
 
+
 ### 4.3. Escher Advanced
+
+This section is intended to provide a more detailed view on the way Escher works and how to create your own custom assets.
+
+#### 4.3.1. Assets basics
+
+I can think of three reasons why one might need to create new assets, for fun not included:
```

Comment:
It's kind of unclear what an asset means to someone who's not familiar with the term in the context of web development.